packages/utils/tool-utils/src/fluidToolRC.ts

```diff
@@ -7,8 +7,8 @@ import fs from "fs";
 import os from "os";
 import path from "path";
 import util from "util";
```

Comment:
Guess biome is recognizing these as node: without the explicit prefix - looks kind of odd without that. Guess we'll get a lint fixup at some point.

Reply:
Yeah there's a lint rule, unicorn/prefer-node-protocol I think. It should be on for the strict config; not sure about recommended.